recipes/r-ukbtools/meta.yaml

```
@@ -0,0 +1,98 @@
+{% set version = '0.11.3' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-ukbtools
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/ukbtools_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/ukbtools/ukbtools_{{ version }}.tar.gz
+  sha256: 031def0cc9e7959173fc016a4c81cf0146eef7248652255009e8e861af4fb7c8
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-xml
+    - r-data.table >=1.12
+    - r-doparallel
+    - r-dplyr
+    - r-foreach
+    - r-ggplot2
+    - r-magrittr
+    - r-purrr
+    - r-readr
+    - r-scales
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+  run:
+    - r-base
+    - r-xml
+    - r-data.table >=1.12
+    - r-doparallel
+    - r-dplyr
+    - r-foreach
+    - r-ggplot2
+    - r-magrittr
+    - r-purrr
+    - r-readr
+    - r-scales
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+
+test:
+  commands:
+    - $R -e "library('ukbtools')"           # [not win]
+    - "\"%R%\" -e \"library('ukbtools')\""  # [win]
+
+about:
+  home: https://kenhanscombe.github.io/ukbtools/
+  license: GPL-2.0-or-later
```

cran meta data # License: GPL-2 matches to
```suggestion
  license: GPL-2.0-only
```